tools/rna_tools/htseq-clip/htseq_clip.xml

```diff
@@ -35,7 +35,7 @@
                 --hce-g $action_type.hce_options.hce_g
                 --hce-q $action_type.hce_options.hce_q
                 $action_type.hce_options.hce_primary
-                --hce-c 4
+                --hce-c 1
```

Comment:
this should use GALAXY_SLOTS I think you want to make it configurable, correct?

Reply:
Yes, I was going to ask how to adjust this for Galaxy ...  :)

Reply:
How to write this here ? I looked in some wrappers and it is always written like:
` --cpus "\${GALAXY_SLOTS:-12}"`
What does the number mean ? Max CPUs supported ?

Reply:
That is bash syntax, it's means 12 if the variable is not set. A default vakue

Reply:
Then I'll give it a go with `--hce-c \${GALAXY_SLOTS:-1}`